toronto/src/pages/Controversy.jsx

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components';
+import { ControversyVote } from '@/components/molecules';
+import { Header, Text } from '@/components/atoms';
+import { useParams, useNavigate } from 'react-router-dom';
+import { useCallback, useEffect, useState } from 'react';
+import { getPostApi, postCommentApi } from '../api/Api';
+
+const Wrapper = styled.div`
+  height: 100vh;
+  padding: 150px 320px;
+  display: flex;
+  flex-direction: column;
+  background-color: #f9fafb;
+`;
+
+const Controversy = () => {
+  const [data, setData] = useState({});
+  const { postId } = useParams();
+  const navigate = useNavigate();
+
+  const getpostData = useCallback(async () => {
+    const postData = await getPostApi(postId);
+    const postTitleData = JSON.parse(postData.data.title);
+    setData({
+      postTitle: postTitleData.postTitle,
+      postContent: postTitleData.postContent,
+      agreeContent: postTitleData.agreeContent,
+      disagreeContent: postTitleData.disagreeContent,
+      image: postData.data.image,
```

Comment:
이 부분은 아래처럼 수정할 수 있을 것 같습니다!

```suggestion
    const { postTitle, postContent, agreeContent, disagreeContent } = JSON.parse(postData.data.title);
    setData({
      postTitle,
      postContent,
      agreeContent,
      disagreeContent,
      image: postData.data.image,
```

Reply:
이걸 놓쳤네요 ㅎㅎㅎ 감사합니다. :)